libcxx/test/std/algorithms/alg.modifying.operations/alg.rotate/rotate.pass.cpp

```diff
@@ -438,9 +438,8 @@ TEST_CONSTEXPR_CXX20 bool test_vector_bool() {
 TEST_CONSTEXPR_CXX20 bool test() {
   types::for_each(types::forward_iterator_list<int*>(), TestIter());
 
-#if TEST_STD_VER >= 11
-  if (TEST_STD_VER >= 23 || !TEST_IS_CONSTANT_EVALUATED)
-    types::for_each(types::forward_iterator_list<std::unique_ptr<int>*>(), TestUniquePtr());
+#if TEST_STD_VER >= 11 && TEST_STD_AT_LEAST_23_OR_RUNTIME_EVALUATED
+  types::for_each(types::forward_iterator_list<std::unique_ptr<int>*>(), TestUniquePtr());
```

Comment:
Oh, IIUC `TEST_STD_AT_LEAST_23_OR_RUNTIME_EVALUATED` is not a preprocess-time condition. (ditto below)

```suggestion
#if TEST_STD_VER >= 11
  if (TEST_STD_AT_LEAST_23_OR_RUNTIME_EVALUATED)
    types::for_each(types::forward_iterator_list<std::unique_ptr<int>*>(), TestUniquePtr());
```

Reply:
I believe you're right, I'll fix it tomorrow. Thanks!

Reply:
Fixed, thanks again. Can't believe I forgot what I introduced in #74186!